post_to_github.sh

```diff
@@ -0,0 +1,9 @@
+rid=$(curl -H "Content-Type: application/json" -H "Authorization: token $1" -X POST -d '{"tag_name":"v'$(cat VERSION)'", "target_commitish":"env/dev-main","name":"v'$(cat VERSION)'","body":"", "draft":false, "prerelease":false}' https://api.github.com/repos/goshippo/shippo-java-client/releases | jq '.id')
```

Comment:
why is this referencing the branch `"target_commitish":"env/dev-main"`?

Reply:
Oversight. It should of course be release